helm/aws-efs-csi-driver-bundle/templates/iam.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
+  name: {{ .Values.clusterID }}-aws-efs-csi-driver-role
```

Comment:
I guess this also becomes the name in AWS IAM.

It differs from `<cluster>-efs-csi-driver-role` naming that capa-iam-operator used. Will we then change capa-iam-operator to delete the objects with the old name, that is, we migrate instead of taking over the resources? (Just want to confirm – makes sense to me in this way.)